src/warnet/graph.py

```diff
@@ -17,3 +24,159 @@ def import_json(infile: Path, outfile: Path, cb: str, ln_image: str):
+        f.write(defaults_yaml_content)
+
+    click.echo(
+        f"Project '{datadir}' has been created with 'network.yaml' and 'node-defaults.yaml'."
+    )
+
+
+def inquirer_create_network(project_path: Path):
+    # Custom network configuration
+    questions = [
+        inquirer.Text(
+            "network_name",
+            message=click.style("Enter your network name", fg="blue", bold=True),
+            validate=lambda _, x: len(x) > 0,
+        ),
+        inquirer.List(
+            "nodes",
+            message=click.style("How many nodes would you like?", fg="blue", bold=True),
+            choices=["8", "12", "20", "50", "other"],
+            default="12",
+        ),
+        inquirer.List(
+            "connections",
+            message=click.style(
+                "How many connections would you like each node to have?",
+                fg="blue",
+                bold=True,
+            ),
+            choices=["0", "1", "2", "8", "12", "other"],
+            default="8",
+        ),
+        inquirer.List(
+            "version",
+            message=click.style(
+                "Which version would you like nodes to run by default?", fg="blue", bold=True
+            ),
+            choices=SUPPORTED_TAGS,
+            default=DEFAULT_TAG,
+        ),
+    ]
+
+    net_answers = inquirer.prompt(questions)
+    if net_answers is None:
+        click.secho("Setup cancelled by user.", fg="yellow")
+        return False
+
+    if net_answers["nodes"] == "other":
+        custom_nodes = inquirer.prompt(
+            [
+                inquirer.Text(
+                    "nodes",
+                    message=click.style("Enter the number of nodes", fg="blue", bold=True),
+                    validate=lambda _, x: int(x) > 0,
+                )
+            ]
+        )
+        if custom_nodes is None:
+            click.secho("Setup cancelled by user.", fg="yellow")
+            return False
+        net_answers["nodes"] = custom_nodes["nodes"]
+
+    if net_answers["connections"] == "other":
+        custom_connections = inquirer.prompt(
+            [
+                inquirer.Text(
+                    "connections",
+                    message=click.style("Enter the number of connections", fg="blue", bold=True),
+                    validate=lambda _, x: int(x) >= 0,
+                )
+            ]
+        )
+        if custom_connections is None:
+            click.secho("Setup cancelled by user.", fg="yellow")
+            return False
+        net_answers["connections"] = custom_connections["connections"]
+
+    custom_network_path = project_path / "networks" / net_answers["network_name"]
+    click.secho("\nGenerating custom network...", fg="yellow", bold=True)
+    custom_graph(
+        int(net_answers["nodes"]),
+        int(net_answers["connections"]),
+        net_answers["version"],
+        custom_network_path,
+    )
+    return custom_network_path
+
+
+@click.command()
+def create_network():
```

Comment:
nice! @willcl-ark had suggested previously we use `warnet init` and `warnet new` for initialising or creating new project directories. this matches cargo's new/init. 

if we did that, we could just use `warnet create` for creating a new network? I prefer the smaller cli command names and I don't envision a user creating anything _but_ a new network.

we could also just have one command for initialising, e.g. `warnet init` and use `warnet create` for creating a network, which would be easy enough to do in this PR, and then we can add `warnet new` later if we feel its needed?

Reply:
What do you think about dropping create / new?

Reply:
id be fine to only have `warnet init`. initially I was thinking `warnet create` would be helpful when you want to manage multiple warnet directories, but starting to feel like no one uses it and it was premature optimisation.

id much rather we have `warnet init` and then have `warnet create` for creating networks.

Reply:
Done in [df2856d](https://github.com/bitcoin-dev-project/warnet/pull/524/commits/df2856deeccb24f68471edc02f02ba415236bd8a)

Reply:
`cargo` has `cargo init` and `cargo create` which IMO make sense. Both make a new project, one in current dir, and one in the dir provided as arg.

No strong opinion though